builder/package.ubuntu-2004

```diff
@@ -21,6 +21,7 @@ fpm \
   -d gcc \
   -d gfortran \
   -d libbz2-1.0 \
+  -d libblas3 \
```

Comment:
R also needs `liblapack3` here (libopenblas includes both BLAS and LAPACK by itself). Or the dev versions: `libblas-dev` and `liblapack-dev`. We've been including the `-dev` packages in the installers since a lot of R packages need them to install from source.

Reply:
Updated to include `libblas-dev` and `liblapack-dev` as dependencies.